src/pages/guides/sellers/marketing-review-guidelines.md

```diff
@@ -55,11 +55,11 @@ For guides and examples of long descriptions, see [product descriptions](product
 
 ## User guide, reference manual, or installation guide
 
-Extension documentation should fulfill the following guidelines:
+Product documentation should fulfill the following guidelines:
 
-*  Must cover all features of the extension.
+*  Must cover all features of the product.
 *  Must not direct users to make purchases on sites other than Marketplace.
-*  If an extension title is added into the documentation, it must not include _Magento_. Use _For Magento_ after the title if needed.
+*  If an product title is added into the documentation, it must not include _Magento_. Use _For Magento_ after the title if needed.
```

Comment:
Please, correct typo "If a product...." 

Please update to _Magento Open Source_ and then _For Magento Open Source

Please  ad also _Adobe Commerce_ and then _For Adobe Commerce_